File: ST/ST.Data.Entities/TrajectReport.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ST.Data.Entities
{
    public class TrajectReport
    {
        public TrajectReport()
        {
            Lines = new List<TrajectReportLine>();
        }

        public List<TrajectReportLine> Lines { get; set; }
        public double TotalDuration { get; set; }
        public double TotalDistance { get; set; }
    }
}

File: SmartTrackApp/app/src/main/java/com/isp/smarttrackapp/entities/TrajectReport.java
package com.isp.smarttrackapp.entities;

import java.util.List;

public class TrajectReport {
    public List<TrajectReportLine> lines;
    public double totalDuration;
    public double totalDistance;

    public List<TrajectReportLine> getLines() {
        return lines;
    }

    public void setLines(List<TrajectReportLine> lines) {
        this.lines = lines;
    }

    public double getTotalDuration() {
        return totalDuration;
    }

    public void setTotalDuration(double totalDuration) {
        this.totalDuration = totalDuration;
    }

    public double getTotalDistance() {
        return totalDistance;
    }

    public void setTotalDistance(double totalDistance) {
        this.totalDistance = totalDistance;
    }
}
